### ISSUE TITLE
User reaches HTTP 500 error page once logged in to GCConnex

### ISSUE THREAD

**Issue by KevinChagnon** [tbs-sct/gcconnex#538]
He errors out only in Newsfeed specifically
User was previously advised to change URL to https://gcconnex.gc.ca/dashboard/  once he logs into GCconnex (or reach the HTTP 500 error page) which works as a temporary solution.

Other users are able to sign in to their own profile on his computer without getting an error.
He gets the same error when trying to sign in on someone else's computer.

Username: Sundeep.Cambow

**Comment by Phanoix** [comment]
Is this still happening after the update this weekend?

**Comment by KevinChagnon** [comment]
Yes it is.  The user gets the following message (confirmed this afternoon):

The gcconnex.gc.ca page isn’t working
gcconnex.gc.ca is currently unable to handle this request.
HTTP ERROR 500

**Comment by KevinChagnon** [comment]
Same as # 625
